app/main.py

```diff
@@ -1,8 +1,25 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    for person_dict in people:
+        Person(name=person_dict["name"], age=person_dict["age"])
+
+    for person_dict in people:
+        current_person = Person.people.get(person_dict["name"])
+        if "wife" in person_dict and person_dict["wife"]:
```

Comment:
```suggestion
        if person_dict.get("wife"):
```
using `get` will simplify the condition